config.yaml

```
@@ -0,0 +1,98 @@
+  file_enabled: true
+  batch_specific: true
+
+filtering:
+  company_filter_enabled: true
+  date_range_filter_enabled: true
+  ciks:
+    - "0000320193"
+  company_names:
+    - "International Business Machines Corp"
+  date_range_start: "2023-01-01"
+  date_range_end: "2023-12-31" 
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)